include/node.hpp

```diff
@@ -519,7 +572,7 @@ class merged_node : public node<merged_node<Left, Right, OutId, InId>, meta::con
     template<std::size_t I>
     [[gnu::always_inline]] constexpr auto
     apply_left(auto &&input_tuple) noexcept {
-        return [&]<std::size_t... Is>(std::index_sequence<Is...>) {
+        return [&]<std::size_t... Is>(std::index_sequence<Is...>) LAMBDA_ALWAYS_INLINE {
```

Comment:
Does this `LAMBDA_ALWAYS_INLINE` make a real-world difference?

Reply:
Yes. :sob: